src/borg/testsuite/repository.py

```diff
@@ -94,8 +91,10 @@ def repo_dump(repository, label=None):
     print()
 
 
-def test_basic_operations(repository):
-    with repository:
+@pytest.mark.parametrize("fixture", ["repository", "remote_repository"])
```

Comment:
For the tests that were previously ran with both a local and remote repo, I decided to use parameterization to accomplish the same thing.